### PR TITLE
Task2699_implement-inputs-transfer-list

### PR DESCRIPTION
**Pull request by nghiatt1509**
Modified simpleTransferListFrame.java to simpleTransferListPage.java
Modified EnhancedTransferListFrame.java to EnhancedTransferListPage.java
Modified simpleTransferList under enhancedTransferListTestSection in TransferListTests.java to enhancedTransferList
Updated URL for simpleTransferListPage in static class
Removed redundant scripts

### CODE REVIEW

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/TransferList/TransferList.java [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import com.epam.jdi.light.elements.common.UIElement;
 import com.epam.jdi.light.elements.complex.ISetup;
 import com.epam.jdi.light.elements.complex.table.CacheAll;
+import com.epam.jdi.light.elements.pageobjects.annotations.WaitAfterAction;
```
Comment (cold-flame) [inline jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/TransferList/TransferList.java:10]: Why u need this import ?

Comment (nghiatt1509) [reply jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/TransferList/TransferList.java]: Removed

**Comment by cold-flame** [comment]
All correct

**Review by cold-flame — APPROVED** [review]

**Review by cold-flame — APPROVED** [review]

**Comment by nghiatt1509** [comment]
Approved by Denis Laptov

**Pull request merged**